src/UWP.Images/UWP.Images.projitems

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>0be54bbb-7772-4289-bd51-1fdbb0cc2446</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>UWP.Images</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)camera.png" />
```

Comment:
How are 2x, 3x, etc handled in UWP?

Reply:
these are the guidelines for images scaling: https://msdn.microsoft.com/library/windows/apps/xaml/hh965325?f=255&MSPPError=-2147217396

but the other resolutions can be added after having a read to publish app, if you are OK?